commands/slash/remove.js

```diff
@@ -26,7 +26,7 @@ const command = new SlashCommand()
       const joinEmbed = new MessageEmbed()
         .setColor(client.config.embedColor)
         .setDescription(
-          "You have to join voice channel first before you can use this command"
+          "You have to a join voice channel before you can use this command"
```

Comment:
I think you meant to switch the a and join 
"You have to join a voice channel before you can use this command"